pkg/operator/startupmonitorreadiness/readiness_checks.go

```diff
@@ -377,3 +379,17 @@ func filterByNodeName(kasPods []corev1.Pod, currentNodeName string) []corev1.Pod
 
 	return filteredKasPods
 }
+
+func getExcludedChecks() []string {
+	return []string{
+		"api-openshift-apiserver-available",
+	}
+}
+
+func appendExcludedChecksToAddress(rawURL string) string {
+	newURL := rawURL
+	for _, excludedCheck := range getExcludedChecks() {
+		newURL = fmt.Sprintf("%s&exclude=%s", newURL, excludedCheck)
```

Comment:
this won't work when `getExcludedChecks` returns more than one item, am I right?

Reply:
It would, as we keep appending stuff to `newURL`, not `rawURL`. 

Tested this in unit-tests:
```
unexpected message "unexpected query params received: verbose=true&exclude=api-openshift-apiserver-available&exclude=more-stuff", expected "/healthz?verbose=true&exclude=api-openshift-apiserver-available\": EOF"
```